azure/dns-records/output.tf

```diff
@@ -0,0 +1,11 @@
+output "a_fqdn" {
+  value = { for k, fqdn in azurerm_dns_a_record.record : k => fqdn.fqdn }
+}
+
+output "cname_fqdn" {
+  value = { for k, fqdn in azurerm_dns_cname_record.record : k => fqdn.fqdn }
+}
+
+output "txt_fqdn" {
+  value = { for k, fqdn in azurerm_dns_txt_record.record : k => fqdn.fqdn }
+}
```

Comment:
Please consider adding descriptions to the outputs. For some examples, please check the `storage-account` and `keyvault` modules.

Reply:
Added